lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -442,7 +442,7 @@ newDBLayer logConfig trace mDatabaseFile = do
               selectWallet wid >>= \case
                   Just _ -> do
                       metas <- selectPendingTxs wid (TxId tid)
-                      let checkStatus = not . any
+                      let checkStatus = any
```

Comment:
:no_mouth:

Reply:
yes, I know - I would put palm_facing emoticon but it is missing ;-)

Reply:
:man_facepalming: :woman_facepalming: